### PR TITLE
System Status feature #175

### PR DESCRIPTION
**Pull request by rdymade**
Feature #175 implemented as described.

### CODE REVIEW

**Comment by rdymade** [comment]
@pfh59 you know whats wrong there? do i need to fix something?

File: src/WHMapper/Pages/Mapper/CustomNode/EveSystemNode.razor
Comment (pfh59) [inline src/WHMapper/Pages/Mapper/CustomNode/EveSystemNode.razor:1]: move this code in EveSystemNode.cs
Create property like Locked property

File: src/WHMapper/Models/Custom/Node/EveSystemNodeModel.cs
Comment (pfh59) [inline src/WHMapper/Models/Custom/Node/EveSystemNodeModel.cs:1]: add a event :  public event Action<EveSystemNodeModel>? OnSystemStatusChanged;
Invoke event on Set SystemStatus (see Locked as work)

File: src/WHMapper/Pages/Mapper/CustomNode/EveSystemNode.cs [hunk 1]
```diff
@@ -26,6 +27,7 @@ public partial class EveSystemNode : ComponentBase
 
         private EveSystemNodeModel _node = null!;
 
+        private HubConnection _hubConnection=null!;
```
Comment (pfh59) [inline src/WHMapper/Pages/Mapper/CustomNode/EveSystemNode.cs:30]: Remove it, and use hub connection on main OverView page when event OnSystemStatusChanged triggering.
It necessary to send this info in real time to other

**Review by pfh59 — APPROVED** [review]
@rdymade, don't forget to sign CLA, I just fix CLA bot
go for merging
Nice job ;-)

**Comment by pfh59** [comment]
recheck

**Comment by rdymade** [comment]
recheck

**Comment by pfh59** [comment]
merged, I hava an issue with CLA system. We will sign it on next PR

**Pull request merged**